src/main/java/core/basesyntax/operations/BalanceOperation.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.operations;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.Map;
+
+public class BalanceOperation implements OperationHandler {
+    @Override

```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line before the method declaration.